worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -586,14 +586,14 @@ private[celeborn] class Worker(
 
   override def exit(exitType: String): String = {
     exitType match {
-      case "DECOMMISSION" =>
+      case "decommission" =>
```

Comment:
Support both upper and lower case?

Reply:
Better change this in a separated pr

Reply:
@AngersZhuuuu, this change is caused by the lowercase change of `HttpUtils#parseUri` which does not change host of worker to uppercase. Therefore I change this in this pull request.